contents/blog/google-optimize-alternatives.md

```diff
@@ -0,0 +1,199 @@
+
+### Companies that use Unbounce
+
+- **Packlane:** Custom packaging and boxes for retailers
+- **Uberflip:** Sales engagement and lead-gen platform
+- **PitchBook:** M&A database and marketing intelligence
+
+## 3. GrowthBook
+
+- **Free tier:** Yes
+- **Transparent pricing:** Yes
+- **Unlimited experiments:** Yes
+- **A/B and multivariate tests:** Yes
+- **Session recording:** No
+- **Product analytics:** No
+- **Feature flags:** Yes
+- **Multi-armed bandit:** No
+- **Visual editor:** Yes
+- **Landing page builder:** No
+- **Best for:** Product and engineering teams
+
+[GrowthBook](www.growthbook.io/) is a popular open-source feature flagging and experimentation platform. It supports A/B and multivariate tests, and integrates with a range of popular analytics tools, such as Google Analytics (UA), Mixpanel, Matomo, and most popular data warehouses.
+
+It offers a basic visual editor so non-technical users can create experiments without writing code. GrowthBook's 'Guardrail' feature can also monitor for unintended second order effects that your potential change might impact, but which you're not actively testing against.
+
+### How much does GrowthBook cost?
+
+GorwthBook offers unlimited traffic (subject to a fair use policy), feature flags and experiments, pricing only on the number of user accounts. Its Starter plan includes three GrowthBook users. Its Pro plan adds advanced permissions, custom fields, and its visual A/B test editor, for just $20 per user, per month.
+
+### Companies that use GrowthBook
+
+- **Deezer:** Music app
+- **Brilliant:** Learning platform
+- **Typeform:** Form and survey platform for businesses
+
+## 4. Instapage
+
+- **Free tier:** No
+- **Transparent pricing:** Partial
+- **Unlimited experiments:** Yes
+- **A/B and multivariate tests:** Yes
+- **Session recording:** No
+- **Product analytics:** No
+- **Feature flags:** No
+- **Multi-armed bandit:** No
+- **Visual editor:** Yes
+- **Landing page builder:** Yes
+- **Best for:** Marketing teams
+
+[Instapage](https://instapage.com/) is a marketing optimization platform built around a landing page builder and experimentation suite. It doesn't support multi-armed bandit testing like Unbounce, but it does include a range of useful features for marketing teams, such as personalized landing pages based on ad creative, AMP page support, loading speed optimizations, and dynamic text replacement for PPC campaigns.
+
+### How much does Instapage cost?
+
+Instapage's self-serve plan costs $299 per month ($199 if billed annually) and includes most of its core features with no usage limits. Its upgrade plans are 'price on application' and add numerous value-add features, such as heatmaps, custom font support, and consulting services.
```

Comment:
What does price on application mean?

Reply:
You have to email them for a quote / get a demo. Might come up with a better way to say this.

Reply:
"Its upgrade plans require a sales call, but add numerous..."